mistralrs-core/src/pipeline/gguf.rs

```diff
@@ -463,7 +482,13 @@ impl Loader for GGUFLoader {
                 model_config,
                 device,
             )?;
-            let cache_engine = CacheEngine::new(model_config, &cache_config, DType::F32, device)?;
+            let cache_engine = CacheEngine::new(
+                model_config,
+                &cache_config,
+                DType::F32,
```

Comment:
This won't work with #1009, can you please add a comment (`TODO` perhaps) to remind me/others to update this?

Reply:
Thank you!